core/src/impl/Kokkos_Core.cpp

```diff
@@ -526,7 +565,7 @@ void parse_command_line_arguments(int& narg, char* arg[],
       --kokkos-disable-warnings      : disable kokkos warning messages
       --kokkos-tune-internals        : allow Kokkos to autotune policies and declare
                                        tuning features through the tuning system. If
-				         left off, Kokkos uses heuristics
+				       left off, Kokkos uses heuristics
```

Comment:
Undo this change

Reply:
Oh thats strange... There must be a `\t` in there because this change aligned it properly.

Reply:
![Screen Shot 2020-11-28 at 8 35 52 PM](https://user-images.githubusercontent.com/6001865/100533457-59c92f80-31b9-11eb-8ebd-1d1aa7b23ea6.png)

Yep.